src/demo-app/toolbar/toolbar-demo.html

```diff
@@ -1,40 +1,58 @@
 <div class="demo-toolbar">
-
   <p>
     <mat-toolbar>
-      <mat-icon class="demo-toolbar-icon">menu</mat-icon>
-      <span>Default Toolbar</span>
+      <button mat-icon-button>
```

Comment:
nit: do we want to keep one `mat-icon` without button in the demo?

Reply:
Good point. I have such plain `<mat-icon>` elements in the demo toolbars below.